TheaterSchedule.BLL/Services/SendDataToGoogleFormService.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using TheaterSchedule.BLL.Interfaces;
+namespace TheaterSchedule.BLL.Services
+{
+    public class SendDataToGoogleFormService:ISendDataToGoogleFormService
+    {
+        private string _baseUrl;
+        private Dictionary<string, string> _field;
+        public Dictionary<string, string[]> _checkbox;
```

Comment:
it's bad practice to maintain state in services. Refactor to avoid storing of state